packages/lesswrong/components/seasonal/PetrovDayButton.tsx

```diff
@@ -116,70 +155,93 @@ const PetrovDayButton = ({classes, refetch}: {
   }
 
   const launch = async () => {
-    if (!currentUser) {
-      return
-    }
-    await updateUser({
-      selector: {_id: currentUser._id},
-      data: { 
-        petrovCodesEnteredDate: new Date (),
-        petrovCodesEntered: launchCode
-      }
-    });
-    refetch()
+    if (!currentUser) return
+    
+    const a = await fetch('/graphql', {
```

Comment:
Why is there a fetch here on the client? Should just be a `useQuery` or `useMutation`